crates/biome_js_type_info/src/type_info/literal.rs

```diff
@@ -0,0 +1,213 @@
+use biome_js_type_info_macros::Resolvable;
+use biome_rowan::Text;
+
+#[derive(Clone, Debug, Default, PartialEq, Resolvable)]
+pub struct BooleanLiteral(bool);
+
+impl BooleanLiteral {
+    #[inline]
+    pub fn as_bool(&self) -> bool {
+        self.0
+    }
+
+    pub fn parse(s: &str) -> Option<Self> {
+        match s.as_bytes().first() {
+            Some(b't' | b'T') => Some(Self(true)),
+            Some(b'f' | b'F') => Some(Self(false)),
```

Comment:
Does our parser ever accept capitalised literals for booleans?

Reply:
Nice catch, it was my misunderstanding. Removed uppercase patterns from the match.